pagopa/io-react-native-cie-pid
# feat: [EUDIW-53] Add `iOS` bridge to get NFC tag reader

## Description
This PR adds iOS bridge to read NFC cards.

## How to test

- start example app using a real iOS device with iOS version >= 13
- Tap on Start Cie Manager
- Insert CIE pin
- Tap on continue to show modal message
- Using your CIE wait for reading
- CIE related data should be printed on UI
